package.json

```diff
@@ -40,44 +40,44 @@
         "@colyseus/monitor": "^0.14.22",
         "@colyseus/schema": "^2.0.4",
         "@colyseus/ws-transport": "^0.14.21",
-        "@parcel/config-default": "2.8.0",
```

Comment:
I'm not sure why you got all these changed, make sure to revert the changes on the package.json and on the .lock above as well

Reply:
The dependencies are the same but whenever I run npm install or update they get reorder (alphabetically, maybe?).

Reply:
anyway, reverting those files